resources/js/Pages/Tasks/Task.vue

```diff
@@ -72,7 +72,6 @@
           :targetProps="['id','name']"
           heading='State'/>
       </section>
-      <p><b>Subject:</b> {{ task.title }}</p>
```

Comment:
We call it title in the creation part. I figured I would remove it and place it where we show the ID, as we can just nicely place that up by the word Task :)

Reply:
Sounds good